react-native/react/constants/update.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+
+import type {TypedAction} from '../constants/types/flux'
+import type {Asset, UpdateType} from '../constants/types/flow-types'
+
+export const showUpdatePrompt = 'update:showUpdatePrompt'
+export type ShowUpdateAction = TypedAction<'update:showUpdatePrompt', showUpdatePromptPayload, UpdateError>
+
+export const registerUpdateListener = 'update:registerUpdateListener'
+export type RegisterUpdateListenerAction = TypedAction<'update:registerUpdateListener', any, any>
+
+export const onCancel = 'update:onCancel'
+export type OnCancelAction = TypedAction<'update:onCancel', void, any>
+export const onSkip = 'update:onSkip'
+export type OnSkipAction = TypedAction<'update:onSkip', any, any>
```

Comment:
The second type parameter here represents the type of the payload in this action. If the payload is nothing it should be `void`.